plugins/woocommerce/src/Internal/EmailEditor/WCTransactionalEmails/WCTransactionalEmails.php

```diff
@@ -30,6 +30,8 @@ class WCTransactionalEmails {
 		'customer_reset_password',
 		'failed_order',
 		'new_order',
+		'customer_pos_completed_order',
```

Comment:
No feature-flag conditional here?

Reply:
Good question - the existing usage only uses this list to filter the supported email types in `WC_Emails` so that the additions here don't affect the overall supported email types. But it's a good idea to also put them behind the same feature flag in case the use cases change. Updated in https://github.com/woocommerce/woocommerce/pull/57573/commits/7e3b770ce329a86628bd4d0caae8aa8d6cbdb126.